src/gurt/misc.c

```diff
@@ -443,15 +443,15 @@ d_rank_list_dump(d_rank_list_t *rank_list, d_string_t name, int name_len)
 	for (i = 0; i < rank_list->rl_nr; i++)
 		width += snprintf(NULL, 0, "%d ", rank_list->rl_ranks[i]);
 	width++;
-	D_ALLOC(tmp_str, width);
+	D_MM_ALLOC(tmp_str, width);
```

Comment:
This is a debugging function, it's not performance critical.

Reply:
ok. Will be more focused...